developer/src/server/build.sh

```diff
@@ -124,20 +124,18 @@ fi
 
 if (( build_keymanweb )); then
   pushd "$KEYMAN_ROOT/web/"
-  ./build.sh --no-minify
+  ./build.sh build
```

Comment:
Keyman Developer used the unminified version of KeymanWeb in the past, to make it easier to debug keyboards in JS if needed (which helped for 'imx' style keyboards in particular). Would this still be helpful now? As in, should we add `--debug` here?

Reply:
At the moment, KMW builds both debug and release configurations at the same time.  The testing pages all use the `--debug` variant too, so it should generally be built during standard `$KEYMAN_ROOT/web/build.sh` calls.  So... `--debug` is unnecessary for producing the non-minified version.  We could add it if you're worried about changing that in the future though, I guess.

I suppose I could have `--debug` disable the release build... but that's such a small fraction of the build time that it's insignificant.  I'm pretty sure each of the `esbuild` calls completes within a second, maybe two at most... and that's the only real point of divergence.

Reply:
It's kinda helpful as a mnemonic to remind readers of the script that Developer is using the unminified files, but not super important (and just keeping it consistent with other platform builds which don't build debug files as a step on the way to release builds).

Reply:
```suggestion
  ./build.sh build --debug
```